migration_steps/validation/validate_db/app/sql/fixed_sql/annual_report_lodging_details.sql

```diff
@@ -0,0 +1,48 @@
+-- annual_report_lodging_details
+DROP TABLE IF EXISTS casrec_csv.exceptions_annual_report_lodging_details;
+
+CREATE TABLE casrec_csv.exceptions_annual_report_lodging_details(
+    deadlinedate text default NULL,
+    datereportlodged text default NULL,
+    lodgedstatus text default NULL,
+    bankstatementdeadlinedate text default NULL,
+    bankstatementsreceived text default NULL
+);
+
+INSERT INTO casrec_csv.exceptions_annual_report_lodging_details(
+    SELECT
+        CAST(NULLIF(account."Followup Date", '') AS date) AS deadlinedate,
```

Comment:
These are all definitely dates rather than datetimes right? :)

Reply:
It says "date" in the mapping spreadsheet, so I'm going with that.